guides/common/modules/proc_running-an-ansible-playbook.adoc

```diff
@@ -0,0 +1,27 @@
+[id="running-an-ansible-playbook-from-{project-context}_{context}"]
+= Running an Ansible playbook from {Project}
+
+You can run an Ansible playbook on a host or host group by executing a remote job in {Project}.
+
+.Prerequisites
+* Ansible plugin in {Project} is enabled.
+* You have configured xref:Configuring_and_Setting_Up_Remote_Jobs_{context}[remote job execution].
+* You have an Ansible playbook ready to use.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs*.
+. Click *Run Job*.
+. In *Job category*, select `Ansible Playbook`.
+. In *Job template*, select `Ansible - Run playbook`.
+. Click *Next*.
+. Select the hosts on which you want to run the playbook.
+. In the *playbook* field, paste the content of your Ansible playbook.
+. Optional: Customize any advanced settings and schedule.
+For more information, see xref:executing-a-remote-job_{context}[].
+. Click *Run on selected hosts*.
```

Comment:
I would suggest something along the lines of "follow the wizzard" + "click *Submit* to run the Ansible playbook on your hosts."

Similar to https://github.com/theforeman/foreman-documentation/pull/2941#discussion_r1555324226 & https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/con_standardizing-content-view-empty-states.adoc?plain=1#L6 Relates to #2490

Reply:
Gandalf is big these days. Noted.